test/Microsoft.AspNetCore.Sockets.Tests/HttpConnectionDispatcherTests.cs

```diff
@@ -336,6 +338,90 @@ public HttpConnectionDispatcherTests(ITestOutputHelper output) : base(output)
             }
         }
 
+        [Fact]
+        public async Task HttpContextFeatureForLongpollingWorksBetweenPolls()
```

Comment:
I'm sure @moozzyk would love how long this test is 😄

Reply:
:D